apps/export/entries/report_exporter.py

```diff
@@ -517,6 +518,8 @@ def _generate_for_levels(
                 self.doc.add_paragraph()
 
             if entries:
+                if is_preview:
+                    [self._generate_for_entry(entry) for entry in entries[:50]]
```

Comment:
```suggestion
                    [self._generate_for_entry(entry) for entry in entries[:PREVIEW_ENTRY_SIZE]]
```
Lets define the constant somewhere on the top.